src/rdkafka.c

```diff
@@ -2531,15 +2531,15 @@ int rd_kafka_consume_start0 (rd_kafka_topic_t *rkt, int32_t partition,
 
 int rd_kafka_consume_start (rd_kafka_topic_t *app_rkt, int32_t partition,
```

Comment:
it's noted in the `rd_kafka_new` docstring that this method is for operating in the simple legacy consumer mode. This isn't noted in the docstring for `rd_kafka_consume_start`. I think some more commenting aimed at distinguishing the two would probably be worthwhile (probably other places too).